client/signup/steps/import-from/wordpress/content-chooser/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { NextButton } from '@automattic/onboarding';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import React from 'react';
+import ActionCard from 'calypso/components/action-card';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { jetpack } from 'calypso/signup/icons';
+import SelectItems from 'calypso/signup/select-items';
+
+import './style.scss';
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+export const ContentChooser: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+
+	return (
+		<div className={ classnames( 'import-layout', 'content-chooser' ) }>
+			<div className={ 'import-layout__column' }>
+				<FormattedHeader
+					align={ 'left' }
+					headerText={ __( 'What would you like to import?' ) }
+					subHeaderText={ __( 'Choose what you would like to import to your new site.' ) }
+				/>
+				<div className={ 'step-wrapper__header-image' }>
+					<img alt="" src="/calypso/images/importer/onboarding-2.svg" />
+				</div>
+			</div>
+			<div className={ 'import-layout__column' }>
+				<div>
+					<ActionCard
+						classNames={ classnames( 'list__importer-option', { 'is-disabled': true } ) }
+						headerText={ __( 'Everything' ) }
+						mainText={ __( "Your site's content, themes, plugins, users and settings" ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'All your site's content, themes, plugins, users and settings' )` *ES Score: 9*